lib/tasks/exchange_rate_importer.rake

```diff
@@ -0,0 +1,76 @@
+# Import TARIC or CDS file manually. Usually for initial seed files.
+namespace :importer do
+  namespace :rates do
+    desc 'Import Average Exchange Rates'
+
+    task import_agv_ex_rates: %i[environment] do
+      average_type = 'average'
+
+      url = 'https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1154375/Average_for_the_year_to_31_March_2023.csv'
```

Comment:
we need to import all the rates from 2020 onwards. This only imports 1 file, It should be 2 files per year